data/set/disjointed.lean

```diff
@@ -5,7 +5,7 @@ Authors: Johannes Hölzl
 
 Disjointed sets
 -/
-import data.set.lattice data.nat.basic
+import data.set.lattice data.nat.basic tactic.wlog
```

Comment:
why?

Reply:
It's because of transitive imports. `tactic.wlog` is no longer imported by the files that this one imports.